src/common/tabs/BaseTabView.ts

```diff
@@ -14,6 +16,9 @@ export default abstract class BaseTabView implements TabView {
     get name(): string {
         return getTabViewName(this.server.name, this.type);
     }
+    get tuple(): TabTuple {
```

Comment:
Can we update the field name here? The name `tuple` doesn't necessarily describe what it is. Maybe something like `urlTypeTuple`?